web/src/routing/routes/UsersAdministration/Users/UsersTable/UsersTable.tsx

```diff
@@ -0,0 +1,91 @@
+import { VFC } from "react";
+
+import { useTheme } from "@mui/material/styles";
+import { Theme, Checkbox } from "@mui/material";
+
+import { Table } from "@components/Table/Table";
+import { ConditionalText } from "@components/ConditionalText/ConditionalText";
+
+import { User } from "@janush-types/user";
+
+const columns = [
+  { name: "checkbox", label: "" },
+  { name: "id", label: "ID" },
+  { name: "email", label: "Email" },
+  { name: "phoneNumber", label: "Phone Number" },
+  { name: "access", label: "Access" },
+  { name: "status", label: "Status" },
+  { name: "lastmodified", label: "Last Modified" },
+];
+
+interface Props {
+  data: User[];
+  onRowClick: (user: User) => void;
+}
+
+const UsersTable: VFC<Props> = ({ data, onRowClick }) => {
+  const { TableHead, TableRow, TableCell, TableBody } = Table;
```

Comment:
That operation happens every re-render, I think we can move it outside of component.